modules/microshift-restoring-backups-auto-recovery.adoc

```diff
@@ -3,36 +3,33 @@
 // * microshift/microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="microshift-restoring-backups_{context}"]
+[id="microshift-restoring-backups-auto-recovery_{context}"]
 = Restoring backups using the auto-recovery feature
 
-You can restore backups after system events that remove or damage required data.
-
-Use the following procedure to restore backups.
+You can restore backups after system events that remove or damage required data. Use the following procedure to restore backups using automatic recovery. Automatic recovery selects the most recent backup and restores it.
```

Comment:
Not sure why we didn't include that detail previously, but it might be useful to at least mention it as it might accumulate old backups and use space.
```suggestion
You can restore backups after system events that remove or damage required data. Use the following procedure to restore backups using automatic recovery. Automatic recovery selects the most recent backup and restores it. Previously auto-recovery restored backup is moved to `PATH/restored`.
```